streams/src/main/java/org/apache/kafka/streams/errors/BrokerNotFoundException.java

```diff
@@ -16,13 +16,13 @@
  */
 package org.apache.kafka.streams.errors;
 
-
 /**
  * Indicates that none of the specified {@link org.apache.kafka.streams.StreamsConfig#BOOTSTRAP_SERVERS_CONFIG brokers}
  * could be found.
  *
  * @see org.apache.kafka.streams.StreamsConfig
  */
+@SuppressWarnings("unused")
```

Comment:
I'm not a big fan of adding this, but I get why we'd do so

Reply:
Yeah. IntelliJ "complaints" about it -- the goal is to get the "green checkmark" :)